client/src/index.css

```diff
@@ -15,10 +14,25 @@ body {
 }
 
 .the-whole-page {
-    top: 0;
-    text-align: center;
     background-color: black;
-    width: 100%;
+    top: 0;
+    width: auto;
+    height: 100%;
+}
+
+.left-panel {
+    display: block;
+    color: white;
+    background-color: #6c757d;
```

Comment:
Darker looks better e.g. #484b4f

Reply:
It does!  I had to fix up the physics panel though.